xmodule/capa/responsetypes.py

```diff
@@ -1,16 +1,10 @@
-#
-# File:   courseware/capa/responsetypes.py
-#
 """
 Problem response evaluation.  Handles checking of student responses,
 of a variety of types.
 
 Used by capa_problem.py
 """
-
 # standard library imports
-
-
 import abc
 # TODO: Refactor this code and fix this issue.
```

Comment:
@farhaanbukhsh It's actually coming from https://github.com/openedx/edx-platform/pull/10986/files#diff-46d349acd0abe955aa35f20425137351a56a2b6865ef5e358c58f81e5932e283 and it refers to a `pylint: disable=attribute-defined-outside-init` which has actually been fixed, so you're right, it can be removed.